crates/relay/src/lib.rs

```diff
@@ -241,13 +245,92 @@ pub mod pallet {
                 reports.insert(vault_id.clone());
             });
 
-            // reward the participant by increasing their SLA
+            // reward the reporter by increasing their SLA
             ext::sla::event_update_vault_sla::<T>(&signer, ext::sla::Action::TheftReport)?;
-
             Self::deposit_event(<Event<T>>::VaultTheft(vault_id, tx_id));
 
             Ok(().into())
         }
+
+        /// Report Vault double payment, providing two fraud proofs (malicious Bitcoin transactions
+        /// and the corresponding transaction inclusion proofs).
+        ///
```

Comment:
```suggestion
        /// This can be used for any multiple of payments, i.e., a vault making two, three, four, etc. payments
        /// by proving just one double payment.
        /// A Vault is fully slashed on committing theft.
        ///
```